docs/.pre-commit-config.yaml

```
@@ -0,0 +1,46 @@
+---
+repos:
+  # import formatter with black configurations
+    - repo: https://github.com/pycqa/isort
```

Consider revising the comment to match the indentation of the line below.
```suggestion
    # import formatter with black configurations
    - repo: https://github.com/pycqa/isort
```